ops/platform-dev/README.md

```diff
@@ -0,0 +1,35 @@
+# Content library on the GKE platform cluster
+
+
+This directory contains the Kubernetes manifests needed to deploy the content library app onto the GKE platform cluster. Note that the Terraform code should all be applied for the respective environment you are trying to deploy before applying these manifests.
+
+
+## Prerequisites
+
+- The GCP service account must be authorized to run on the platform GKE cluster. A cluster administrator must perform this whitelisting action. If you are a cluster admin, see [this document for instructions](https://github.com/adhocteam/adhoc-gke-dev-platform/blob/main/docs/administration/authorizing_a_service_account.md) on how to authorize a GCP service account with the GKE cluster.
+
+
+- The Terraform code must all be deployed
+- The `SECRET_ENV_FILE` must be populated with valid data the app can start with
```

Comment:
`SECRET_ENV_FILE`? how does this work

Reply:
I just realized this should not exist. I was misunderstanding something and had it corrected but didnt update the docs